bindings/bind_py_structure_manager.cc

```diff
@@ -124,11 +124,16 @@ namespace rascal {
     return py_cluster;
   }
 
-  //! Bind iterator and ClusterRef for Order >= 2
+  /**
+   * Bind iterator and ClusterRef for Order >= 2
+   *
+   * @pre Order >= 2
+   */
   template <typename StructureManagerImplementation, size_t Order>
   decltype(auto) add_iterator(
       py::module & m,
       PyClusterRef<StructureManagerImplementation, Order - 1> & py_cluster) {
```

Comment:
Before this `PyClusterRef` was compiled with `Order=1` resulting in a compiler buffer overflow when accessing the `LayerByOrder`